From 94b185e26f5f5258484c1c7d3f257c18dc56debb Mon Sep 17 00:00:00 2001
From: icg_intg <icg_intg@intel.com>
Date: Mon, 5 Aug 2019 11:03:03 +0800
Subject: [PATCH 2351/2367] isys & psys package lib2600b0 for commit-id dbb2a92

FW_Manifest_ID: 057337c8527b4baca2ecd17a469a291edabac0de

FW_Build_Branch: stable_iot_gp1_ivi_20190410

FW_Build_Topic: auto_merge_queue

library version: commit-id dbb2a92

FW Bundle patch number of this release :  423507

FW tag : scci_stable_iot_gp1_ivi_20190410_20190804_2009

Change-Id: Ic2dba092c4959ac14dbf4883eadc0d93ead44303
Depends-on: I70d4513a16a36aa29c51b08307c6edb0f15bd8b6
Depends-on: Ifaced0b48ca1a49917d782e559523c0c4a442431
Tracked-On: #PKT-2935
Signed-off-by: icg_intg <icg_intg@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
Signed-off-by: Ong Yang Yang <yang.yangx.ong@intel.com>
---
 drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h          | 2 +-
 .../pci/intel/ipu4/ipu4-css/lib2600/config/isys/subsystem_bxtB0.mk     | 3 +++
 .../lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h       | 2 +-
 3 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
index ca869f6..b2d4abc 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20191007
+#define IA_CSS_FW_PKG_RELEASE  0x20190731
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/config/isys/subsystem_bxtB0.mk b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/config/isys/subsystem_bxtB0.mk
index da14203..c4549f3 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/config/isys/subsystem_bxtB0.mk
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600/config/isys/subsystem_bxtB0.mk
@@ -46,6 +46,9 @@ WA_HSD1209062354		 = 1
 # WA to disable clock gating for the devices in the CSI receivers needed for using the mipi_pkt_gen device
 WA_HSD1805168877		 = 0
 
+# WA to match the NofSOF & NofEOF when resuming a stream to prevent fps drop
+WA_HSD1507026600         = 1
+
 # Support IBUF soft-reset at stream start
 SOFT_RESET_IBUF_STREAM_START_SUPPORT = 1
 
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
index ca869f6..b2d4abc 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20191007
+#define IA_CSS_FW_PKG_RELEASE  0x20190731
-- 
2.7.4

